<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplicationGUIClass</class>
 <widget class="QWidget" name="ApplicationGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Polygon Partition</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ApplicationGUI.qrc">
    <normaloff>:/ApplicationGUI/PolygonPartition.ico</normaloff>:/ApplicationGUI/PolygonPartition.ico</iconset>
  </property>
  <widget class="DrawingArea" name="drawingArea_test">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>640</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="DrawingArea" name="polygonDrawingArea">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>22</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="qrb_areaProperties">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>400</y>
     <width>221</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Area properties</string>
   </property>
   <widget class="QLabel" name="lbl_minAreaOfPart">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_polygonArea">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_polygonAreaLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Polygon area:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_minAreaOfPartLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Minimum area:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_maxAreaOfPartLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Maximum area:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_maxAreaOfPart">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frm_slider">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="RangeSlider" name="sld_areaOfPart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>14</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="sld_minPosLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sld_maxPosLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="qrb_meshProperties">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>140</y>
     <width>221</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Mesh properties</string>
   </property>
   <widget class="QPushButton" name="btn_apply_angle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reset_angle">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_meshAngle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mesh angle:</string>
    </property>
   </widget>
   <widget class="DrawingArea" name="meshAngleDrawingArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>200</width>
      <height>130</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_meshAngle">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="suffix">
     <string>Â°</string>
    </property>
    <property name="maximum">
     <double>179.990000000000009</double>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="qrb_polygonProperties">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Polygon properties</string>
   </property>
   <widget class="QPushButton" name="btn_apply">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reset">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_numOfSides">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Num of sides:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_numOfSides">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>42</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
   </widget>
  </widget>
  <widget class="DrawingArea" name="drawingArea_test_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>640</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="DrawingArea" name="drawingArea_test_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>640</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="DrawingArea" name="drawingArea_test_4">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>640</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_saveResults">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>440</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save Results</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawingArea</class>
   <extends>QLabel</extends>
   <header>drawingarea.h</header>
  </customwidget>
  <customwidget>
   <class>RangeSlider</class>
   <extends>QSlider</extends>
   <header>rangeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ApplicationGUI.qrc"/>
 </resources>
 <connections/>
</ui>
